src/openeo_grass_gis_driver/actinia_processing/config.py

```diff
@@ -1,20 +1,26 @@
 # -*- coding: utf-8 -*-
 import os
+import configparser
+from pathlib import Path
 
 __license__ = "Apache License, Version 2.0"
-__author__ = "Sören Gebbert"
-__copyright__ = "Copyright 2018, Sören Gebbert, mundialis"
-__maintainer__ = "Soeren Gebbert"
-__email__ = "soerengebbert@googlemail.com"
+__author__ = "Sören Gebbert, Carmen Tawalika"
+__copyright__ = "Copyright 2018-2021, Sören Gebbert, mundialis"
+__maintainer__ = "mundialis"
 
 
-class Config:
-    # Settings for docker swarm image
+# config can be overwritten by mounting *.ini files into folders inside
+# the config folder.
+DEFAULT_CONFIG_PATH = "config"
+CONFIG_FILES = [str(f) for f in Path(
+    DEFAULT_CONFIG_PATH).glob('**/*.ini') if f.is_file()]
+GENERATED_CONFIG = DEFAULT_CONFIG_PATH + '/openeo-grassgis-driver.cfg'
```

Comment:
`*.cfg` or `*.ini` ?

Reply:
I don't mind much, it was taken from [here](https://github.com/mundialis/actinia-module-plugin/blob/main/actinia_module_plugin/resources/config.py#L35). I would prefer consistency but if you have a strong opinion I can change it?

Reply:
No strong opinion here. From scratch I would have used identical file extensions for initial and runtime config files. However, regarding maintenance it makes sense to keep this code part in sync with actinia. You decide.

Reply:
Ok I leave it then.